docs/resource/k8s.md

```diff
@@ -211,3 +211,10 @@ A CronJob creates Jobs on a repeating schedule.
 <!-- endsemconv -->
 
 [DocumentStatus]: https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/document-status.md
+
+## Well-Known Labels
+
+There are certain Kubernetes [Well-Known Labels and Annotations](https://kubernetes.io/docs/reference/labels-annotations-taints/) that SHOULD align to OpenTelemetry resource attributes. Labels are typically converted to existing OpenTelemetry attributes automatically using an OpenTelemetry Collector component like the [Kubernetes Attribute Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor).
+
+* The Well-Known Label `app.kubernetes.io/name` SHOULD align to the OpenTelemetry resource attribute `service.name`.
```

Comment:
There's a [convention about `service.name`](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/resource#service) which is declared stable:
> MUST be the same for all instances of horizontally scaled services. If the value was not specified, SDKs MUST fallback to `unknown_service:` concatenated with [`process.executable.name`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/process.md#process), e.g. `unknown_service:bash`. If `process.executable.name` is not available, the value MUST be set to `unknown_service`.

I'm not sure whether we can "alter" or "amend" a stable attribute definition like proposed here.

Reply:
Perhaps we can say this is allowed as it doesn't modify or amend the convention itself, but is related to something something _external_ (k8s labels) and is an implementation (in k8s) recommendation?

Alternately, could this be organized or put somewhere else?